testing/postgresql-cron/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: G.J.R. Timmer <gjr.timmer@gmail.com>
+# Maintainer:
+pkgname=postgresql-cron
```

Comment:
We prefer to follow the upstream name where possible, and that’s pg_cron, not postgresql-cron.